test/extensions/filters/http/rbac/rbac_filter_integration_test.cc

```diff
@@ -125,6 +128,7 @@ TEST_P(RBACIntegrationTest, Denied) {
   response->waitForEndStream();
   ASSERT_TRUE(response->complete());
   EXPECT_EQ("403", response->headers().getStatusValue());
+  EXPECT_THAT(waitForAccessLog(access_log_name_), testing::HasSubstr("RBAC "));
```

Comment:
can we have a test where the details return additional data?

Reply:
added tests for details.